Fixes flashlight eyes

QualityVan opened the pull request:
Fixes #26413, flashlight eyes now contain actual flashlights. Can't toggle them while they're inside you, though.

Core0verload committed:
Refactors picture frames to use wall frames system

BeeSting12 committed:
removes flashbang's space

QualityVan committed:
Fixes armlasers breaking on retract

Profakos committed:
Prevents fultons from transporting items out of inventories

and an init fix

coiax committed:
Fixes languages being understood over radio

Fixes #26306.
Fixes #26192.

:cl: coiax
fix: Fixed people understanding languages over the radio when they
shouldn't.
/:cl:

Commit:
Updates space ruins to use TGM format and most of them to use template_noop areas and turfs where suitable.

Cyberboss committed:
Fixes proximity checkers being moved by shuttles

Cyberboss committed:
Fix current_ticket.initiator not being set when a client relogs

Cyberboss committed:
Fixes supermatter not spawning teslas

MrsTonedOne committed:
Revert "Ghosts call Moved() after Move() (#26183)"

This reverts commit 0adf5f64a7ae81476e534ea5d77d34b2d7d67c2a.

Cyberboss committed:
Copies the loaded titelscreen into the rsc

XDTM committed:
Fixes gold and light pink slime extracts

MrsTonedOne committed:
fixes a proc cache to be more obvious it's a proc cache

QualityVan committed:
Fixes flashlight eyes

QualityVan committed:
Fixes flashlight eyes

ChangelingRain commented:
you appear to have nerfed flashlight eyes!
would you like to try again?

ChangelingRain commented on code/modules/surgery/organs/organ_internal.dm:
```diff
@@ -878,17 +878,33 @@
 	flash_protect = 2
 	tint = INFINITY
+	var/obj/item/device/flashlight/eyelight/left_eye
+	var/obj/item/device/flashlight/eyelight/right_eye
```
You don't need two eyes here.

QualityVan commented:
I'm not sure I follow? Unless you mean the old version previously using 15 for its range and 1 for its power, in which case oops. Not entirely sure how to make two lights be as strong as a single one, so maybe just cut the pair?

QualityVan committed:
jesus that's a lotta light

QualityVan committed:
jesus that's a lotta light

octareenroon91 commented:
Maybe just drive this with one flashlight

ChangelingRain commented on code/modules/surgery/organs/organ_internal.dm:
```diff
@@ -878,17 +878,24 @@
-	M.set_light(M.light_range -15, M.light_power - 1)
+	eye.on = FALSE
+	eye.update_brightness(M)
+	eye.loc = src
```
eye.forceMove(src)

ChangelingRain commented on code/modules/surgery/organs/organ_internal.dm:
```diff
@@ -878,17 +878,24 @@
+	if(!eye)
+		eye = new /obj/item/device/flashlight/eyelight()
+	eye.on = TRUE
+	eye.loc = M
```
eye.forceMove(M)

QualityVan committed:
really though this tells you how often i use flashlight eyes

Cheridan committed:
Merge pull request #26388 from QualityVan/armlasers

Fixes armlasers breaking on retract

QualityVan committed:
Fixes fuel checks being too early in weldering airlocks

Core0verload committed:
Cleans up some copypasted src and accidently commited changes

coiax commented on code/game/objects/items/devices/flashlight.dm:
```diff
@@ -428,3 +428,11 @@
+	brightness_on = 15
+	flashlight_power = 1
+	flags = CONDUCT | DROPDEL
+	actions_types = list()
```
This is a hack, we *really* should just make organs inside the person. Not your fault.

Commit:
Automatic changelog compile, [ci skip]

ChangelingRain committed:
Fixes #25029

lzimann committed:
Merge pull request #26417 from MrStonedOne/patch-350

fixes a proc cache to be more obvious it's a proc cache

lzimann committed:
Merge pull request #26415 from XDTM/GrandmasterSlimer

Fixes gold and light pink slime extracts

tgstation-server committed:
Automatic changelog generation for PR #26415 [ci skip]

lzimann committed:
Merge pull request #26411 from Cyberboss/patch-54

Fixes supermatter not spawning teslas

lzimann committed:
Merge pull request #26409 from Cyberboss/ProxyShuttleMovement

Fixes proximity checkers being moved by shuttles

lzimann committed:
Merge pull request #26405 from Jalleo/SpaceRuinsUpdate

Updates space ruins TGM format and noop usage

lzimann committed:
Merge pull request #26403 from coiax/fix-language-radio

Fixes languages being understood over radio

tgstation-server committed:
Automatic changelog generation for PR #26403 [ci skip]

lzimann committed:
Merge pull request #26402 from Profakos/cleanit

Prevents fultons from transporting items out of inventories

Cyberboss committed:
Conveyor movement refactor/fix (#26408)

* Refactors conveyor movement

* Fixes effects being affected by conveyors

* Fixes jetpacks not stopping conveyors

* Fixes /mob/dead being affected by conveyors

* Fixes butterflys being affected by conveyors

tgstation-server committed:
Automatic changelog generation for PR #26408 [ci skip]

optimumtact committed:
Merge pull request #26337 from Core0verload/picture_frame

Refactors picture frames to use wall frames system

lzimann committed:
Merge pull request #26422 from QualityVan/doorwelding

Fixes fuel checks being too early in weldering airlocks

optimumtact committed:
Merge pull request #26412 from MrStonedOne/revert-ghost-call_moved

Revert "Ghosts call Moved() after Move() (#26183)"

Commit:
Automatic changelog compile, [ci skip]

MMMiracles committed:
poly's ghost no longer haunts the space of cerestation round-start (#26365)

* :ok:

* travis stop

ChangelingRain committed:
Secrets panel logging is better (#26418)

coiax committed:
Refactors ear damage into ear organs (#26044)

* Refactors ear damage into ear organs

:cl: coiax
add: Centcom would like to inform all employees that they have ears.
add: Adds "ear" organs to all carbons. These organs store ear damage and
deafness. A carbon without any ears is deaf. Genetic
deafness functions as before.
/:cl:

- `ear_damage` and `ear_deaf` vars removed from /mob.
- All mobs have a `can_hear` proc, which returns TRUE always except for
carbons.
- Carbons need to have an ear organ that has 0 `deaf` var.

- Explanation of how ear damage works is in the code, it hasn't been
changed from previously. Deafness is applied in number of Life ticks
until you regain hearing, while damage is long team, heals slower, and
when high enough, can cause flashbangs to make you go permamently deaf,
as before.

- Wearing earmuffs halves the healing time of deafness, and promotes
healing long term ear damage, as before. Earmuffs now have a secondary
flag HEALS_EARS, which currently only they own.

* Changes how soundbang deafness works slightly

* Ear organ icon

* Code review I

* Makes fully healing carbons not dependent on having a dna and species

* Gives monkeys and aliens ears

* Whoops

* Split organs into seperate files

* Tweaks.

* Un-removes brain damage lines

* Moved procs onto /mob for ear stuff

* Massages things into compiling

* Replacement of spam_flag with world.time tracker

tgstation-server committed:
Automatic changelog generation for PR #26044 [ci skip]

Cyberboss committed:
Heinious spawn (#26332)

coiax committed:
Fixes runtime when carbon mobs have no earwear (#26442)

My bad.

Cyberboss committed:
Minor refactors (#26305)

obj/on_log changed to atom/on_log. Boolean parameter in
No admin irc message now checked in client/Del instead of mob/Logout
Removed an empty New()
1 -> TRUE
removed a potential crash in mob/Login

lzimann committed:
Removes rogue bold tag

BeeSting12 committed:
fixes compile issues

lzimann committed:
Merge pull request #26427 from ChangelingRain/lighterfix

Greyscale lighters no longer color the flame

lzimann committed:
Merge pull request #26414 from Cyberboss/TitleFix

Copies the loaded titlescreen into the rsc

Cyberboss committed:
Quite possibly the smallest microoptimization here (#26407)

lzimann committed:
Merge pull request #26410 from Cyberboss/FixesTicketReconnection

Fix current_ticket.initiator not being set when a client relogs

lzimann committed:
Merge pull request #26376 from BeeSting12/flashbangs

Grammar edit for flashbangs tip

Commit:
Automatic changelog compile, [ci skip]

optimumtact committed:
Merge pull request #26454 from tgstation/lzimann-roguetag

Removes rogue bold tag in player panel

MrsTonedOne committed:
fixes the server (#26489)

QualityVan committed:
Rebasing

QualityVan committed:
Merge branch 'flashlighteyes' of https://github.com/QualityVan/tgstation into flashlighteyes

# Conflicts:
#	code/modules/surgery/organs/organ_internal.dm

QualityVan commented:
holy shit that's not how rebasing works
let's try this again

QualityVan commented:
Okay I think it worked this time
